templates/components/common/navigation.html

```diff
@@ -43,79 +34,125 @@
+                        </mask>
+                        <g mask="url(#mask0_2973_15350)">
+                        <rect width="24" height="24" fill="#272727"/>
+                        </g>
+                    </svg>
+                    <span class="js-nav-user-action-text c-action-text">{{lang 'common.account'}}</span>
                 </a>
             {{else}}
                 <a 
-                    class="js-nav-user-action"
+                    class="js-nav-user-action c-nav-page-action"
                     href="{{urls.auth.login}}"
                     aria-label="{{lang 'common.login'}}"
                 >
-                    {{lang 'common.login'}}
+                    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
+                        <mask id="mask0_2973_15350" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="24" height="24">
+                        <path d="M20 21V19C20 17.9391 19.5786 16.9217 18.8284 16.1716C18.0783 15.4214 17.0609 15 16 15H8C6.93913 15 5.92172 15.4214 5.17157 16.1716C4.42143 16.9217 4 17.9391 4 19V21M12 11C14.2091 11 16 9.20914 16 7C16 4.79086 14.2091 3 12 3C9.79086 3 8 4.79086 8 7C8 9.20914 9.79086 11 12 11Z" stroke="black" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+                        </mask>
+                        <g mask="url(#mask0_2973_15350)">
+                        <rect width="24" height="24" fill="#272727"/>
+                        </g>
+                    </svg>
+                    <span class="js-nav-user-action-text c-action-text">{{lang 'common.login'}}</span>
                 </a>
-                {{#if settings.account_creation_enabled}}
-                    <span>{{lang 'common.or'}}</span>
-                    <a 
-                        class="js-nav-user-action"
-                        href="{{urls.auth.create_account}}"
-                        aria-label="{{lang 'common.sign_up'}}"
-                    >
-                        {{lang 'common.sign_up'}}
-                    </a>
-                {{/if}}
             {{/if}}
         </li>
-        <li>
-            <a 
-                class="js-nav-user-action js-cart-preview"
-                data-dropdown="cart-preview-dropdown"
-                data-options="align:right"
-                href="{{urls.cart}}"
-                aria-label="{{lang 'cart.nav_aria_label'}}"
-            >
-                <span>{{lang 'common.cart'}}</span>
-                <span class="js-count-pill js-cart-quantity"></span>
-            </a>
-
-            <div 
-                class="js-dropdown-menu js-cart-preview-dropdown"
-                aria-hidden="true"
-            ></div>
-        </li>
+        <a class="u-hidden@medium c-menu-toggle" href="#" data-mobile-menu-toggle="menu">
+            <svg class="c-menu-icon" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
```

Comment:
Not following BEM rules - the class should be `c-menu__icon` not `c-menu-icon` unless `menu-icon` is a separate component